render/gui/src/main/java/net/flintmc/render/gui/windowing/WindowRenderer.java

```diff
@@ -42,6 +45,8 @@
    */
   boolean isIntrusive();
```

Comment:
From what I can tell, this can be inferred based on whether `getIntrusiveScreens()` is empty or not